src/test/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImplTest.java

```diff
@@ -33,6 +33,7 @@
     public void setup() {
         project = (ProjectInternal) ProjectBuilder.builder().build();
         dockerBuildImageProvider = createBuildDockerImage(project);
+        createBuildDockerImageAlternative(project);
```

Comment:
You don't want to set up this extra task for all tests, but only for your specific test.

You can do any setup you want in your test as long as it happens before you call methods on `composeConvention`

Reply:
Okey. Thanks.